src/Renderer.js

```diff
@@ -1,45 +1,66 @@
+export const FORME_RECT = 0;
+export const FORME_CIRCLE = 1;
+export const FORME_TRIANGLE = 2;
+
 const render = (canvas, ctx, entity) => {
     //TODO simple implementation, change me with entity form and color
     const relative = compute(canvas, entity);
-    ctx.beginPath();
-    ctx.rect(relative.x, relative.y, relative.width, relative.height);
     ctx.fillStyle = entity.color;
-    ctx.fill();
+    ctx.beginPath();
+    if(entity.forme === FORME_RECT) {
```

Comment:
Seems a lot of code for a simple if (x == 0) else if x == 1...
maybe you can try:
var forms = {
0: () => ctx.rect(relative.x, relative.y, relative.width, relative.height);
1: () => ...
2: () => ...
} 
forms[entity.forme]();
It might work with javascript closure magic !